Sources/Target/Windows/X86/ThreadX86.cpp

```diff
@@ -51,8 +55,9 @@ ErrorCode Thread::readCPUState(Architecture::CPUState &state) {
                          CONTEXT_EXTENDED_REGISTERS; // SSE registers.
 
   BOOL result = GetThreadContext(_handle, &context);
```

Comment:
I don't like it :(

On Wed, Apr 13, 2016 at 11:42 AM, walter erquinigo <notifications@github.com
> wrote:

> In Sources/Target/Windows/X86/ThreadX86.cpp
> <https://github.com/facebook/ds2/pull/273#discussion_r59602989>:
>
> > @@ -51,8 +55,9 @@ ErrorCode Thread::readCPUState(Architecture::CPUState &state) {
> >                           CONTEXT_EXTENDED_REGISTERS; // SSE registers.
> >
> >    BOOL result = GetThreadContext(_handle, &context);
>
> I'd just write `if (!GetThreadContext...) {
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/273/files/b72ddba7246b0e66911a55da1efe908c08c10d0f#r59602989>
>



-- 
Stephane Sezer